extensions/wikia/CreateNewWiki/js/WikiBuilder.js

```diff
@@ -606,6 +606,7 @@ define('ext.createNewWiki.builder', ['ext.createNewWiki.helper', 'wikia.tracker'
 				},
 				callback: function (res) {
 					throbberWrapper.stopThrobbing();
```

Comment:
I don't remember what we decided about throbber itself, are we leaving the old one or should we change it to spinner?